test/draw/RasterMonochromeTest.ooc

```diff
@@ -98,6 +98,16 @@ RasterMonochromeTest: class extends Fixture {
 			image referenceCount decrease()
 			image2 referenceCount decrease()
 		})
+		this add("color monochrome from unsigned types", func {
+			image := RasterMonochrome new(IntVector2D new(10, 10))
+			value32: UInt32 = 128
+			value64: UInt64 = 128
+			image[0, 0] = ColorMonochrome new(value32)
+			image[0, 1] = ColorMonochrome new(value64)
+			expect(image[0, 0] y as Int, is equal to(value32 as Int))
```

Comment:
We should probably add that since this happens in other places. Will you or shall I?